### ISSUE TITLE
learning_rate

### ISSUE THREAD

**Issue by yunsikchoi** [yunsikchoi/CS231n#87]
![learning rate](https://user-images.githubusercontent.com/36406676/53091420-42293580-3555-11e9-9a24-911e173fe32f.PNG)
